internal/core/items/text.rs

```diff
@@ -722,7 +722,7 @@ impl TextInput {
     }
 
     fn copy(self: Pin<&Self>, window: &WindowRc) {
-        window.set_clipboard_text(self.selected_text());
+        self.with_selected_text(|text| window.set_clipboard_text(text));
```

Comment:
What i mean is that 

```suggestion
        let (anchor, cursor) = self.selection_anchor_and_cursor();
        window.set_clipboard_text(&self.text()[anchor..cursor])
```

Doesn't really require a separate function.   (we could even write it in one line if selection_anchor_and_cursor returned a range :-) )

Also, on linux (with the exception of vscode 👹),  CTRL+C doesn't change the clip board if nothing is selected.  So one might do `if anchor != cursor` before calling set_clipboard_text

Reply:
Ah yes, sure, if that works with `SharedString` then that's best. I'll make that change then and get rid of the helper function to get the selected text.